lib/instance.js

```diff
@@ -563,8 +564,9 @@ instance.prototype.connect = function (client) {
 	});
 
 	client.on('instance_order', function(old_index, new_index) {
-		var instances = self.instances;
+		var instances;
 		if (instances !== undefined) {
```

Comment:
This if will never go through, since you just defined instances as undefined :P

Reply:
Funny ... I was just looking through this code last night.  Doing this ends up being more difficult than I expected.  The drag order is using a numeric index for the sort order but ultimately the keys in the array its sorting are the instance IDs.  I haven't found an elegant way to make that all work yet.

Reply:
the same reason why my time shifted to something else ;-)

Reply:
I'm going with punt to v3